src/components/Map/Map.tsx

```diff
@@ -18,7 +19,7 @@ function Map({
   hideLegend = false,
   hideInstructions = false,
   hideLegendTitle = false,
-  onClick = () => {},
+  onClick = noop,
```

Comment:
Without this, `USACountyMap` ended up re-rendering every time the homepage rendered since `onClick` changed every time.

Reply:
oof yeah, another nasty trap.